dts/arm/silabs/xg29/xg29.dtsi

```diff
@@ -0,0 +1,446 @@
+			clocks = <&cmu CLOCK_USART0 CLOCK_BRANCH_PCLK>;
+			status = "disabled";
+		};
+
+		usart1: usart@50060000 {
+			compatible = "silabs,gecko-usart";
+			reg = <0x50060000 0x400>;
+			interrupts = <18 0>, <19 0>;
+			interrupt-names = "rx", "tx";
+			clocks = <&cmu CLOCK_USART1 CLOCK_BRANCH_PCLK>;
+			status = "disabled";
+		};
+
+		burtc0: burtc@50064000 {
+			compatible = "silabs,gecko-burtc";
+			reg = <0x50064000 0x4000>;
+			interrupts = <23 0>;
+			interrupt-names = "burtc";
+			clocks = <&cmu CLOCK_BURTC CLOCK_BRANCH_EM4GRPACLK>;
+			status = "disabled";
+		};
+
+		i2c1: i2c@50068000 {
+			compatible = "silabs,gecko-i2c";
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			reg = <0x50068000 0x4000>;
+			interrupts = <33 0>;
+			interrupt-names = "i2c1";
+			clocks = <&cmu CLOCK_I2C1 CLOCK_BRANCH_PCLK>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		dcdc: dcdc@50094000 {
+			compatible = "silabs,series2-dcdc";
+			reg = <0x50094000 0x4000>;
+			interrupts = <8 0>;
+			interrupt-names = "dcdc";
+			status = "disabled";
+		};
+
+		eusart1: eusart@500b4000 {
+			compatible = "silabs,gecko-spi-eusart";
+			reg = <0x500B4000 0x4000>;
+			interrupts = <68 0>, <69 0>;
+			interrupt-names = "rx", "tx";
+			clocks = <&cmu CLOCK_EUSART1 CLOCK_BRANCH_EM01GRPCCLK>;
+			status = "disabled";
+		};
+
+		rtcc0: rtcc@58000000 {
+			compatible = "silabs,gecko-stimer";
+			reg = <0x58000000 0x4000>;
+			interrupts = <15 0>;
+			interrupt-names = "rtcc";
+			clocks = <&cmu CLOCK_RTCC CLOCK_BRANCH_RTCCCLK>;
+			clock-frequency = <32768>;
+			prescaler = <1>;
+			status = "disabled";
+		};
+
+		wdog0: wdog@58018000 {
+			compatible = "silabs,gecko-wdog";
+			reg = <0x58018000 0x4000>;
+			peripheral-id = <0>;
+			interrupts = <49 0>;
+			interrupt-names = "wdog0";
+			clocks = <&cmu CLOCK_WDOG0 CLOCK_BRANCH_WDOG0CLK>;
+			status = "disabled";
+		};
+
+		adc0: adc@5a004000 {
+			compatible = "silabs,gecko-iadc";
+			reg = <0x5a004000 0x4000>;
+			interrupts = <54 0>;
+			interrupt-names = "iadc0";
+			clocks = <&cmu CLOCK_IADC0 CLOCK_BRANCH_IADCCLK>;
+			status = "disabled";
+			#io-channel-cells = <1>;
+		};
+
+		i2c0: i2c@5a010000 {
+			compatible = "silabs,gecko-i2c";
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			reg = <0x5a010000 0x4000>;
+			interrupts = <32 0>;
+			interrupt-names = "i2c0";
+			clocks = <&cmu CLOCK_I2C0 CLOCK_BRANCH_LSPCLK>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		eusart0: eusart@5a040000 {
+			compatible = "silabs,gecko-spi-eusart";
+			reg = <0x5A040000 0x4000>;
+			interrupts = <20 0>, <21 0>;
+			interrupt-names = "rx", "tx";
+			clocks = <&cmu CLOCK_EUSART0 CLOCK_BRANCH_EUSART0CLK>;
+			status = "disabled";
+		};
+
+		se: semailbox@4c000000 {
+			compatible = "silabs,gecko-semailbox";
+			reg = <0x4c000000 0x1000>;
+			interrupts = <0 3>, <1 3>, <2 3>;
+			interrupt-names = "SETAMPERHOST", "SEMBRX", "SEMBTX";
+			status = "disabled";
+		};
```

Comment:
Not a big deal, these devices declarations are sorted by base addresses. I believe it is easier to group them by family.

Reply:
Reordered EUSART0 and I2C0, since they were separated from EUSART1 and I2C1, but kept the rest of the peripherals sorted by address.